src/infrastructure/repository/RequestRepositoryFile.ts

```diff
@@ -185,8 +185,9 @@ export class RequestRepositoryFile implements RequestRepository {
 
   private async writeResponseBody(request: Request, response: Response) {
     const bodyFilePath = await this.getResponseBodyFilePath(request);
+    const buffer = Buffer.from(response.body, 'binary');
```

Comment:
I'm not really sure whether `Buffer.from` is necessary here as the `responseType` is already set to buffer